Header header

uint32 timestamp

uint32 channel

uint32 width
uint32 height

float32[] inphase
float32[] quadrature

bool[] validity